src/app/pages/viewing/responses-list/responses-list.component.css

```
@@ -21,51 +27,98 @@
   margin-left: 50px;
 }
 
-.close-btn {
+/* Button component */
+.hamburger {
+  position: absolute;
+  height: 6em;
+  width: 2em;
+  top: calc(2em);
```

Why is there a calc without any variables?
```suggestion
  top: 2em;
```